### DISCUSSION TITLE
Is there a way to change the message on password prompt??

### DISCUSSION THREAD

**Discussion by niteshk9894** [coder/code-server#5977] [Q&A] [answered]
Wanted to know if there is a way to change the message in password prompt. The message means below image highlighted one.

![image](https://user-images.githubusercontent.com/86285273/213448978-09307cb9-a900-4129-9caf-59156b672d7e.png)

Thanks for the help in advance.

**Comment by jsjoeio** [comment, marked as answer]
I don't think so but you can open an issue to request that!

**Reply by niteshk9894** [reply]
have opened it https://github.com/coder/code-server/issues/5980.
